scripts/plugin_list.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+import pathlib
+import re
+
+import packaging.version
+import requests
+import tabulate
+
+FILE_HEAD = r"""Plugins List
+============
+
+The plugins are listed automatically from PyPI.
+Only PyPI projects that match "pytest-\*" are considered plugins.
+Packages classified as inactive are also excluded.
+
+"""
+DEVELOPMENT_STATUS_CLASSIFIERS = (
+    "Development Status :: 1 - Planning",
+    "Development Status :: 2 - Pre-Alpha",
+    "Development Status :: 3 - Alpha",
+    "Development Status :: 4 - Beta",
+    "Development Status :: 5 - Production/Stable",
+    "Development Status :: 6 - Mature",
+    "Development Status :: 7 - Inactive",
+)
+
+
+def iter_plugins():
+    regex = r">([\d\w-]*)</a>"
+    response = requests.get("https://pypi.org/simple")
+    for match in re.finditer(regex, response.text):
+        name = match.groups()[0]
+        if not name.startswith("pytest-"):
+            continue
+        response = requests.get(f"https://pypi.org/pypi/{name}/json")
+        if not response.ok:
+            continue
+        info = response.json()["info"]
+        if "Development Status :: 7 - Inactive" in info["classifiers"]:
+            continue
+        for classifier in DEVELOPMENT_STATUS_CLASSIFIERS:
+            if classifier in info["classifiers"]:
+                status = classifier[22:]
+                break
+        else:
+            status = "NA"
+        requires = "NA"
+        if info["requires_dist"]:
+            for requirement in info["requires_dist"]:
+                if requirement == "pytest" or "pytest " in requirement:
+                    requires = requirement
+                    break
+        releases = response.json()["releases"]
+        for release in sorted(releases, key=packaging.version.parse, reverse=True,):
```

Comment:
I think it shouldn't be reversed. The column says `last release`, where it actually shows the first release date. So my guess would be that it's the `reverse=True` fault.

Reply:
Thanks for noticing the wrong displayed information. I fixed the issue! I forgot to add a break statement inside the if.

Reply:
I still see the invalid data. Or does the generated documentation provided by @nicoddemus has to be somehow rebuilt manually?

Reply:
OK, now I see it updated. All good, thanks.